ama-sessions/session-2.md

```diff
@@ -0,0 +1,110 @@
+The time estimates are our best estimate for watching the videos and completing the exercises. That said, every student consumes content at a different pace so pls take this into account when building in time in your schedule. Once you complete a few of the lessons, you'll be able to best gauge how these times compare to your own pace
+
+#### I'm pretty sure my question is not related to this schoolarship, but I'm just being curious. Are Google developers forced to use Google Products? I've seen in the past some Microsoft Developers using Internet Explorer in some of their Web Developing courses which was hilarious.
+I really don't know the answer to that and can't speak for Googlers (I work for Udacity). But I bet many of them like using Google products, which is why they wanted to work there.
+
+#### Can you recommend any way of contributing that isn't as time-sensitive as bug squashing, or will the community interaction be weighted less than the actual quality of the projects submitted ?
+We completely understand that many of you have full-time work, school and life commitments and appreciate that this scholarship program will fit into your other responsibilities. In terms of community interaction, we encourage you to contribute when you can and works for your schedule. It's more about the quality of your contributions and engagement than quantity.
+
+#### I don't have the full pre-reqs for the Mobile Web Specialist Nanodegree. Should I ask to switch to Front End?
+Good question! We intended for our pre-reqs to be guideposts. Not everyone will have all of them. And you may need to do some learning on the side, but based on your application, we felt like this program would be the best challenge for you. If you have *no* web dev experience and find the material too difficult, please do let us know.
+
+#### Are you full-time hired to spend time on Slack? That sounds like a cool job.
+I do have a cool job em! I head up our scholarships team at Udacity (check out all our scholarship programs at udacity.com/scholarships). So I don't spend all my time on Slack, but we do have full time community managers (like anaisbrg ) and program managers (like Chaitra) and they have awesome jobs too!
+
+#### Do you plan any other scholarship for europe ? I think I saw something about data.
+We will be updating our scholarships page (udacity.com/scholarships) with all new offering so definitely check the site for any updates and fill out the information requested under the “notify me” section so you receive emails when new programs become available
+
+#### Are we earning a pretty Badge at least if we don't succeed to the second phase? It's a good 3 months course full of insight and we might want to show it off in our LinkedIn without sounding like we failed to get into the Nano-degree. ```
```

Comment:
Please fix text formatting (answers looks the same as questions) and we will be ready to merge it.